great_expectations/core/metric_function_types.py

```diff
@@ -0,0 +1,38 @@
+import enum
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MetricFunctionTypes(enum.Enum):
```

Comment:
Can we use the `str` mixin here like: `class MetricFunctionTypes(enum.Enum)` so we don't need to access `.value`?

Reply:
@NathanFarmer I am not sure -- potentially, but it would be subject of a separate change, because this change is purely module refactoring.